flake.nix

```diff
@@ -307,6 +307,7 @@
                     cargo-udeps
                     pkgs.parallel
                     pkgs.just
+                    pkgs.linuxKernel.packages.linux_5_10.perf
```

Comment:
Is this a remnant from performance testing?

Reply:
Ha, yes.